cmd/generate-registry-metadata/README.md

```
@@ -0,0 +1,57 @@
+    "path": "ipi/install/ipi-install-ref.yaml",
+    "owners": {
+        "approvers": ["user1", "user2"],
+        "reviewers": ["user3"]
+    }
+}
+```
+
+The `path` field is the component's path relative to the registry root, and `owners` is the full parsed OWNERS config from the component's directory.
+
+## Flags
+| Flag | Default | What it controls |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)